.gitattributes

```diff
@@ -1,3 +1,11 @@
+# ~~~
+# Copyright (c) 2018-2023 The Khronos Group Inc.
```

Comment:
Just 'Copyright <dates> <owner>' is fine. The © symbol literally just means "Copyright" and has been redundant for decades in international law. to the best of my knowledge. It's less clear that ASCII '(c)' means anything at all.